packages/adaptive_scaffold/example/lib/adaptive_layout_demo.dart

```diff
@@ -0,0 +1,102 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:adaptive_scaffold/adaptive_scaffold.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(
+    const MaterialApp(
+      title: 'Adaptive Layout Example',
+      home: _MyHomePage(),
+    ),
+  );
+}
+
+class _MyHomePage extends StatelessWidget {
+  const _MyHomePage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // Define the list of destinations to be used within the app.
+    const List<NavigationDestination> destinations = <NavigationDestination>[
+      NavigationDestination(
+          label: 'Inbox', icon: Icon(Icons.inbox, color: Colors.black)),
+      NavigationDestination(
+          label: 'Articles',
+          icon: Icon(Icons.article_outlined, color: Colors.black)),
+      NavigationDestination(
+          label: 'Chat',
+          icon: Icon(Icons.chat_bubble_outline, color: Colors.black)),
+      NavigationDestination(
+          label: 'Video',
+          icon: Icon(Icons.video_call_outlined, color: Colors.black)),
+    ];
+
+    // AdaptiveLayout has a number of slots that take SlotLayouts and these SlotLayouts' configs take maps of Breakpoints to SlotLayoutConfigs.
```

Comment:
Comments should all be wrapped at 80 columns. We're more strict about that than with code lines.